### ISSUE TITLE
Create New Landing Page

### ISSUE THREAD

**Issue by karrla** [buildly-release-management/buildly-react-template#77]
**Description**
Create a new Insights landing page at 
https://insights.buildly.io/

**Tasks**

- [x] Embed code below for the new landing page
- [x] Add an "Insights Login" (https://insights.buildly.io/login) in the footer next to the Buildy Services and Buildly Support links

**Embed this code for the page**
```
<div style="position: relative; width: 100%; height: 0; padding-top: 56.2225%;
 padding-bottom: 48px; box-shadow: 0 2px 8px 0 rgba(63,69,81,0.16); margin-top: 1.6em; margin-bottom: 0.9em; overflow: hidden;
 border-radius: 8px; will-change: transform;">
  <iframe loading="lazy" style="position: absolute; width: 100%; height: 100%; top: 0; left: 0; border: none; padding: 0;margin: 0;"
    src="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAE36D_wnUI&#x2F;watch?embed" allowfullscreen="allowfullscreen" allow="fullscreen">
  </iframe>
</div>
<a href="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAE36D_wnUI&#x2F;watch?utm_content=DAE36D_wnUI&amp;utm_campaign=designshare&amp;utm_medium=embeds&amp;utm_source=link" target="_blank" rel="noopener">Buildly Insights Landing Page</a> by Jeffrey Akita

```

**Comment by RadhikaPPatel** [comment]
@karrla @glind After using the embedded code provided above it shows as a video. Do we want it this way only?

![image](https://user-images.githubusercontent.com/70138493/154417670-b6b77e2f-3eb0-469f-a883-483df48c3cb4.png)

**Comment by karrla** [comment]
is this better?

`<div style="position: relative; width: 100%; height: 0; padding-top: 56.2225%;
 padding-bottom: 48px; box-shadow: 0 2px 8px 0 rgba(63,69,81,0.16); margin-top: 1.6em; margin-bottom: 0.9em; overflow: hidden;
 border-radius: 8px; will-change: transform;">
  <iframe loading="lazy" style="position: absolute; width: 100%; height: 100%; top: 0; left: 0; border: none; padding: 0;margin: 0;"
    src="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAE36D_wnUI&#x2F;view?embed" allowfullscreen="allowfullscreen" allow="fullscreen">
  </iframe>
</div>
<a href="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAE36D_wnUI&#x2F;view?utm_content=DAE36D_wnUI&amp;utm_campaign=designshare&amp;utm_medium=embeds&amp;utm_source=link" target="_blank" rel="noopener">Buildly Insights Landing Page</a> by Jeffrey Akita`

**Comment by karrla** [comment]
try new code

`<div style="position: relative; width: 100%; height: 0; padding-top: 56.2225%;
 padding-bottom: 48px; box-shadow: 0 2px 8px 0 rgba(63,69,81,0.16); margin-top: 1.6em; margin-bottom: 0.9em; overflow: hidden;
 border-radius: 8px; will-change: transform;">
  <iframe loading="lazy" style="position: absolute; width: 100%; height: 100%; top: 0; left: 0; border: none; padding: 0;margin: 0;"
    src="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAE36D_wnUI&#x2F;view?embed" allowfullscreen="allowfullscreen" allow="fullscreen">
  </iframe>
</div>
<a href="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAE36D_wnUI&#x2F;view?utm_content=DAE36D_wnUI&amp;utm_campaign=designshare&amp;utm_medium=embeds&amp;utm_source=link" target="_blank" rel="noopener">Buildly Insights Landing Page</a> by Jeffrey Akita`

**Comment by RadhikaPPatel** [comment]
Merged to production branch in PR #80
